server/svix-server/src/core/types/mod.rs

```diff
@@ -218,11 +218,28 @@ fn validate_limited_str(s: &str) -> std::result::Result<(), ValidationErrors> {
     }
 }
 
+fn validate_non_empty_limited_str(s: &str) -> std::result::Result<(), ValidationErrors> {
```

Comment:
Maybe just fix `validate_limited_str` to not accept empty? (it really shouldn't)